### PR TITLE
manifest: Update ZBOSS libraries to v3.8.0.1

### PR DESCRIPTION
**Pull request by tomchy**
Change manifest file to use sdk-nrfxlib revision with the newest ZBOSS libraries.

Signed-off-by: Tomasz Chyrowicz <tomasz.chyrowicz@nordicsemi.no>

### CODE REVIEW

**Review by sebastiandraus — APPROVED** [review]

File: doc/nrf/shortcuts.txt
Comment (greg-fer) [inline doc/nrf/shortcuts.txt:57]: We need to update this beauty before this PR is merged.

**Review by FilipZajdel — APPROVED** [review]

File: doc/nrf/shortcuts.txt
Comment (tomchy) [reply doc/nrf/shortcuts.txt:57]: Updates

**Review by greg-fer — CHANGES_REQUESTED** [review]
The documentation html build is broken, so do not merge this PR yet.

**Review by greg-fer — APPROVED** [review]
The FTP is working. Unblocking and approving.

**Comment by LuDuda** [comment]
Please update the manifest: 6a8dc21a9019af2d6495c8a7006b6816f5830b35

**Comment by sebastiandraus** [comment]
Nrfxlib revision updated in manifest, one commit squashed.

**Pull request merged**
